run.js

```diff
@@ -0,0 +1,25 @@
+import parse from './lib/index.js';
```

Comment:
This looks like test code and is not required as part of this PR. Please remove this file from PR.

Reply:
removed